www/js/search.js

```diff
@@ -647,25 +647,41 @@ module.exports = {
     }
   },
 
-  loadCeremony(ceremonyID) {
+  async loadCeremony(ceremonyID) {
     const $shabadList = this.$shabad || document.getElementById('shabad');
     $shabadList.innerHTML = '';
-    banidb.loadCeremony(ceremonyID).then(rowsDb => {
-      const rows = rowsDb.map(rowDb => {
-        let row = rowDb;
+    $shabadList.dataset.bani = '';
+    try {
+      const rowsDb = await banidb.loadCeremony(ceremonyID);
+      const rows = await Promise.all(
+        rowsDb.map(async rowDb => {
```

Comment:
there's no `await` used  in this this call back. doesn't need to be `async`.

Reply:
Thanks, fixed it.